cmd/notary/main.go

```diff
@@ -11,20 +10,25 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
+	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/version"
 )
 
-const configFileName string = "config"
-const defaultTrustDir string = ".notary/"
-const defaultServerURL = "https://notary-server:4443"
-const idSize = 64
+const (
+	configFileName   string = "config"
+	defaultTrustDir  string = ".notary/"
+	defaultServerURL        = "https://notary.docker.io"
```

Comment:
well u can make it a compile time var

On Mon, Aug 10, 2015 at 5:18 PM, Diogo Mónica <notifications@github.com>
wrote:

> In cmd/notary/main.go
> <https://github.com/docker/notary/pull/180#discussion_r36700302>:
>
> >  	"github.com/docker/notary/pkg/passphrase"
> >  	"github.com/docker/notary/version"
> >  )
> >
> > -const configFileName string = "config"
> > -const defaultTrustDir string = ".notary/"
> > -const defaultServerURL = "https://notary-server:4443"
> > -const idSize = 64
> > +const (
> > +	configFileName   string = "config"
> > +	defaultTrustDir  string = ".notary/"
> > +	defaultServerURL        = "https://notary.docker.io"
>
> I'm ok with removing the default. It's just been useful for testing
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/notary/pull/180/files#r36700302>.
>